packages/types/src/index.ts

```diff
@@ -77,6 +77,12 @@ export abstract class Connector {
    */
```

Comment:
The above doc comment should be amended, as "This property must be defined while the connector is active." is no longer true.